### ISSUE TITLE
youTubeExtraction.getVideoStreams() is null

### ISSUE THREAD

**Issue by LiuDongCai** [Commit451/YouTubeExtractor#35]
It was always good，But not working recently

**Comment by marcoRS** [comment]
@Jawnnypoo  The extraction no longer seems to be working with the current pattern. Running the Unit tests gives the following error:

```
java.lang.RuntimeException: java.lang.Exception: failed to find pattern "ytplayer.config\s*=\s*(\{.*?\});
	at io.reactivex.internal.util.ExceptionHelper.wrapOrThrow(ExceptionHelper.java:46)
	at io.reactivex.internal.observers.BlockingMultiObserver.blockingGet(BlockingMultiObserver.java:93)
	at io.reactivex.Single.blockingGet(Single.java:2700)
```

**Comment by Jawnnypoo** [comment]
It is now fixed with `5.0.0`
